contrib/mesos/pkg/scheduler/plugin.go

```diff
@@ -430,7 +430,7 @@ func (q *queuer) Run(done <-chan struct{}) {
 			}
 
 			pod := p.(*Pod)
-			if pod.Spec.NodeName != "" {
+			if _, ok := pod.ObjectMeta.Annotations[annotation.BindingHostKey]; ok {
```

Comment:
perhaps we can encapsulate this field & map lookup in a private package func? (and also test for Annotations != nil before the k/v lookup) with some documentation for the func that explains why we're not using pod.Spec.NodeName. otherwise it's pretty confusing when comparing schedulers.